onnxruntime/test/providers/cpu/ml/svmclassifier_test.cc

```diff
@@ -47,22 +53,28 @@ TEST(MLOpTest, SVMClassifierMulticlassSVC) {
 TEST(MLOpTest, SVMClassifierMulticlassLinearSVC) {
   OpTester test("SVMClassifier", 1, onnxruntime::kMLDomain);
 
-  std::vector<float> dual_coefficients = {-1.55181212e-01f, 2.42698956e-01f, 7.01893432e-03f, 4.07614474e-01f, -3.24927823e-02f, 2.79897536e-04f, -1.95771302e-01f, -3.52437368e-01f, -2.15973096e-02f, -4.38190277e-01f, 4.56869105e-02f, -1.29375499e-02f};
+  std::vector<float> dual_coefficients = {-1.55181212e-01f, 2.42698956e-01f, 7.01893432e-03f, 
+                                          4.07614474e-01f, -3.24927823e-02f, 2.79897536e-04f, 
+                                          -1.95771302e-01f, -3.52437368e-01f, -2.15973096e-02f, 
+                                          -4.38190277e-01f, 4.56869105e-02f, -1.29375499e-02f};
   std::vector<int64_t> classes = {0, 1, 2, 3};
   std::vector<float> rho = {-0.07489691f, -0.1764396f, -0.21167431f, -0.51619097f};
   std::vector<float> kernel_params = {0.001f, 0.f, 3.f};  //gamma, coef0, degree
 
-  std::vector<float> X = {1.f, 0.0f, 0.4f, 3.0f, 44.0f, -3.f, 12.0f, 12.9f, -312.f, 23.0f, 11.3f, -222.f, 23.0f, 11.3f, -222.f, 23.0f, 3311.3f, -222.f, 23.0f, 11.3f, -222.f, 43.0f, 413.3f, -114.f};
+  std::vector<float> X = {1.f, 0.0f, 0.4f, 3.0f, 44.0f, -3.f,
+                          12.0f, 12.9f, -312.f, 23.0f, 11.3f, -222.f,
+                          23.0f, 11.3f, -222.f, 23.0f, 3311.3f, -222.f, 
+                          23.0f, 11.3f, -222.f, 43.0f, 413.3f, -114.f};
   std::vector<int64_t> predictions = {1, 0, 1, 1, 1, 0, 1, 0};
   std::vector<float> scores = {
-      -0.227270544f, 0.332829535f, -0.279307127f, -0.518262208f,
-      10.1172562f, -0.282575697f, -16.1046638f, 0.659568906f,
-      -0.996162534f, 4.30999184f, -0.232234091f, -0.707304120f,
-      -2.45976996f, 8.87092972f, -3.76557732f, -6.76487541f,
-      -2.45976996f, 8.87092972f, -3.76557732f, -6.76487541f,
-      798.446777f, -98.3552551f, -1166.80896f, 144.001923f,
-      -2.45976996f, 8.87092972f, -3.76557732f, -6.76487541f,
-      92.7596283f, 3.99134970f, -151.693329f, 1.44020212f};
+      -0.227271f, 0.33283f, -0.279307f, -0.518262f,
```

Comment:
dumb question: where is the numeric gap from?

Reply:
Not sure I understand the question, the figures look the same except I reduce the number of digits.

Reply:
yes, you're changing the precision, this means numeric compliance definition changed, right?

Reply:
I replaced by the previous version. I might have made that change because there was a warning tell me that there were too many digits for float precision.